powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/fido2/PowerAuthRegistrationProvider.java

```diff
@@ -75,7 +75,7 @@ public RegistrationChallenge provideChallengeForRegistration(String userId, Stri
 
     @Override
     @Transactional
-    public RegistrationChallenge provideChallengeForRegistrationChallengeValue(String applicationId, String challengeValue) throws Fido2AuthenticationFailedException {
+    public RegistrationChallenge findRegistrationChallengeByValue(String applicationId, String challengeValue) throws Fido2AuthenticationFailedException {
```

Comment:
Out of the scope of this PR: May this method be `@Transacional(readonly = true)`?

Reply:
I will review the `@Transactional` annotations, good point.